decidim-proposals/spec/services/decidim/proposals/proposal_serializer_spec.rb

```diff
@@ -148,6 +149,79 @@ module Proposals
             expect(serialized).to include(answer: expected_answer)
           end
         end
+
+        context "with rich text proposal body" do
+          let(:image) { "<img src=\"logo.png\" #{alt_attribute} width=\"407\">" }
+          let(:alt_attribute) { "alt=\"Logo alt attribute\"" }
+          let(:body_content) do
+            <<~TEXT
+              <h2>This is my "heading 2" title</h2>
+              <p>A "normal" description below Heading 2</p>
+              <p><br></p>
+              <h3>Now this is my "heading 3"</h3>
+              <p><br></p>
+              <ul>
+              <li>This is my first option</li>
+              <li>This is my second option</li>
+              <li>This is my third option</li>
+              </ul>
+              <p><br></p>
+              <p>And below an uploaded image</p>
+              <p>#{image}</p>
+              <p><br></p>
+              <p><code>Here is code block</code></p>
+            TEXT
+          end
+          let(:body) do
+            {
+              "en" => body_content,
+              "machine_translation" => {
+                "es" => body_content,
+                "ca" => body_content
+              }
+            }
+          end
+
+          it "serializes the body without HTML tags" do
+            expected_body = <<~TEXT
```

Comment:
Just a side comment here that you can also do the chomp here by adding it at the end of the line, i.e. `<<~TEXT.chomp`.

That way don't need to do `.chomp` at the expectations.